bigbluebutton-html5/imports/ui/components/app/service.js

```diff
@@ -46,7 +49,18 @@ function subscribeFor(collectionName) {
 };
 
 function onError(error, result) {
+
   // console.log("OnError", error, result);
+  console.log('OnError', error, result);
+  callServer('userLogout');
+
+  Auth.clearCredentials(() => {
+    document.location = Storage.get('logoutURL');
```

Comment:
Why here you are using `document.location` and on the lines below you are using `window.location.href`?